src/graphql/event/network.rs

```diff
@@ -84,7 +84,7 @@ impl NetworkThreat {
 
     /// The last time the event was seen in string within the representable
```

Comment:
The comment need to be modified: `The last time ...` -> `The end time ...`

Reply:
I agree with @msk’s suggestion. I hope Octoaide will reflect the change accordingly.